app/services/analytics_events.rb

```diff
@@ -1009,12 +1009,26 @@ def saml_auth_request(
     )
   end
 
-  def user_registration_2fa_additional_setup
-    track_event('User Registration: Additional 2fa setup')
+  # @param [Hash] result
+  # Tracks when the the user has selected and submitted additional two factor auth methods on user registration 
+  def user_registration_2fa_additional_setup(result)
+    track_event('User Registration: Additional 2FA setup', result)
   end
```

Comment:
please expand out the items inside the `result` hash (it's why we have keyword params in all the other methods in this file)